### PR TITLE
Feature: Allow users to update their threads

### PR DESCRIPTION
**Pull request by Mukulsingh27**
### Issue
- #9 

### Demo

https://github.com/Mukulsingh27/threads-lite/assets/54891099/8bf5bfd0-25d9-4260-ad1b-ebfc382c56c7

### CODE REVIEW

File: client/src/components/profile/profile.scss [hunk 1]
```diff
@@ -92,7 +92,8 @@
 		border: none;
 
 		&:hover {
-			background-color: #007bff;
+			background-color: #4570fb;
```
Comment (shreyasikhar) [inline client/src/components/profile/profile.scss:95]: If the color is used at multiple places, i would suggest to use scss variables.

File: client/src/components/profile/timeline.scss [hunk 2]
```diff
@@ -130,12 +135,27 @@ $blue-500: #688afd;
...
 		}
 	}
 
+	&__thread-edited {
+		display: none;
+		position: absolute;
+		top: 0;
+		right: 0;
+		color: $grey-300;
+		font-size: 12px;
```
Comment (shreyasikhar) [inline client/src/components/profile/timeline.scss:154]: Avoid using `px` unit everywhere, instead use `rem` or `em`. This change will help users when the user's viewport is zoomed in/out.

File: client/src/components/profile/Timeline.js [hunk 3]
```diff
@@ -20,25 +22,70 @@ const Timeline = ({ thread, hideUnnecessaryElements }) => {
...
 	});
 
-	const handleDelete = (quoteId) => {
+	// Update graphql mutation
+	const [updateQuote, { loading: updateLoader }] = useMutation(UPDATE_QUOTE, {
+		onCompleted: (data) => {
+			console.log(data);
+		},
+		onError: (error) => {
+			console.log(error);
```
Comment (shreyasikhar) [inline client/src/components/profile/Timeline.js:31]: Better to use `console.error()` or `console.warning()` here, what do you think?

**Review by shreyasikhar — APPROVED** [review]
Good work 💯 

I have added few minor feedbacks. 

Also, if you can add icons for edit and delete instead of labels for the threads, that will be great.

File: client/src/components/profile/profile.scss (hunk 1 above)
Comment (Mukulsingh27) [reply client/src/components/profile/profile.scss:95]: Sure, Will change that.

File: client/src/components/profile/timeline.scss (hunk 2 above)
Comment (Mukulsingh27) [reply client/src/components/profile/timeline.scss:154]: Thanks for the suggestion, will implement it.

File: client/src/components/profile/Timeline.js (hunk 3 above)
Comment (Mukulsingh27) [reply client/src/components/profile/Timeline.js:31]: Sure, I will implement it.

**Pull request merged**